addons/l10n_in_pos/static/src/money_details_popup_in.js

```diff
@@ -0,0 +1,22 @@
+/** @odoo-module */
+
+import { useState } from "@odoo/owl";
+import { MoneyDetailsPopup } from "@point_of_sale/app/utils/money_details_popup/money_details_popup";
+import { patch } from "@web/core/utils/patch";
+
+patch(MoneyDetailsPopup.prototype, {
+    setup() {
+        super.setup(...arguments);
+        const indiaMoneyDetails = Object.entries(this.state.moneyDetails)
```

Comment:
```suggestion
        if(this.pos.company.country_id?.code == "IN"){
            const indiaMoneyDetails = Object.entries(this.state.moneyDetails)
```

I would check country code before the logic